pkg/kubectl/resource_printer.go

```diff
@@ -1021,32 +957,37 @@ func printIngress(ingress *extensions.Ingress, w io.Writer, options PrintOptions
 		return err
 	}
 
-	// Lay out all the rules on separate lines.
-	extraLinePrefix := ""
-	if options.WithNamespace {
-		extraLinePrefix = "\t"
-	}
-	for _, rules := range hostRules {
-		if rules.HTTP == nil {
-			continue
-		}
-		_, err := fmt.Fprintf(w, "%s\t%v\t", extraLinePrefix, rules.Host)
-		if err != nil {
-			return err
-		}
-		if _, err := fmt.Fprint(w, appendLabelTabs(options.ColumnLabels)); err != nil {
-			return err
+	// Lay out all the rules on separate lines if use wide output.
```

Comment:
kubectl get output should always be single-line. We can always print more in describe.

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md#output-conventions

cc @bprashanth